pyroomacoustics/transform/stft.py

```diff
@@ -31,7 +31,7 @@ class STFT(object):
     transform (optional) : str
```

Comment:
I noticed that the doc doesn't render well. The argument name for optional arguments is missing. See [here](https://pyroomacoustics.readthedocs.io/en/latest/pyroomacoustics.realtime.stft.html).
I think the syntax should be

    argument_name: type, optional
        The description of the argument

Reply:
I'll change in this PR